edge-modules/MetricsCollector/Settings.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace MetricsCollector
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Configuration;
+    using Newtonsoft.Json;
+
+    public class Settings
+    {
+        static readonly Lazy<Settings> DefaultSettings = new Lazy<Settings>(
```

Comment:
why is this lazy and static?

Reply:
It is static so it can be accessed without an instance (i.e. Settings.Current.<var>). It is lazy so if the user does not make calls to Settings, it does not read from the environment vars.

Reply:
It is immediately called by ```public static Settings Current```, so it will resolve anyway.

Reply:
It shouldn't be lazy then. @philipktlin  do you agree?

Reply:
Resolving as we should remove lazy across all test modules in one PR.